dashboard/modules/metrics/metrics_head.py

```diff
@@ -233,14 +237,40 @@ def _create_default_grafana_configs(self):
             shutil.rmtree(grafana_config_output_path)
         os.makedirs(os.path.dirname(grafana_config_output_path), exist_ok=True)
         shutil.copytree(GRAFANA_CONFIG_INPUT_PATH, grafana_config_output_path)
+
+        # Overwrite grafana's dashboard provisioning directory based on env var
+        dashboard_provisioning_path = os.path.join(
+            grafana_config_output_path, "provisioning", "dashboards"
+        )
+        os.makedirs(
+            dashboard_provisioning_path,
+            exist_ok=True,
+        )
+        dashboards_path = (
+            self._grafana_dashboard_output_dir
+            if self._grafana_dashboard_output_dir
+            else os.path.join(grafana_config_output_path, "dashboards")
```

Comment:
Maybe this should be calculated when assigning self._grafana_dashboard_output_dir